contexts/selected-bots-context.tsx

```
@@ -0,0 +1,205 @@
+"use client"
+
+import { Button } from "@/components/ui/button"
+import type { FormattedBotData } from "@/lib/types"
+import { ExternalLink } from "lucide-react"
+import { createContext, ReactNode, useContext, useState } from "react"
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)